Example/Authenticator/View Controllers/Connection/ConnectViewController.swift

```diff
@@ -24,28 +24,23 @@ import UIKit
 
 final class ConnectViewController: BaseViewController {
     private lazy var completeView = CompleteView(state: .processing, title: l10n(.processing))
-
+    private var viewModel: ConnectViewModel
```

Comment:
Initialize viewModel here, it's used only here, so remove it from child controllers